### PR TITLE
修改mip.config模版

### PR DESCRIPTION
**Pull request by wupengFEX**
修改mip.config模版

### CODE REVIEW

File: boilerplate/mipcustom.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        {
+            "accid": "",
+            "title": "定制化MIP示例页面"
+        }
+        </script>
+    </mip-custom>
+    <script src="https://mipcache.bdstatic.com/static/v1/mip.js"></script>
+    <script src="https://mipcache.bdstatic.com/static/v1/mip-mustache/mip-mustache.js"></script>
+    <script src="https://mipcache.bdstatic.com/static/v1/mip-custom/mip-custom.js"></script>
+</body>
```
Comment (pearlwang1106) [inline boilerplate/mipcustom.html]: u missed mip-fixed.js

File: boilerplate/mipcustom.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        }
+    </style>
+</head>
+<body>
+    <h3 class="mip-custom-title">以下内容展示的是定制化 MIP 内容</h3>
+    <mip-custom>
+        <script type="application/json">
+        {
+            "accid": "",
+            "title": "定制化MIP示例页面"
```
Comment (pearlwang1106) [inline boilerplate/mipcustom.html]: title need encode

Comment (wupengFEX) [reply boilerplate/mipcustom.html]: done

File: boilerplate/mipcustom.html (hunk 1 above)
Comment (wupengFEX) [reply boilerplate/mipcustom.html]: done

**Pull request merged**